INSTALL.md

```diff
@@ -101,8 +102,13 @@ echo "addnode=dnsseed.hush.quebec" >> ~/.hush/hush.conf
 ```
 
 ## Run a HUSH Node
+It can take over 8 hours to fully sync.
+```ssh
```

Comment:
This is not quite correct, if there is an installed `hushd` in the $PATH, it will use that instead of the just-compiled hushd in the current directory.